docs/UserGuide.md

```diff
@@ -180,10 +182,15 @@ Examples:
 - `filter c/Transport`
 - `filter ft/2022-03 c/Transport`
 
-_This is the result of executing the command(s) above:_
+The expense list before executing the `filter` command looked as follows:
+![Filter](images/Sample_Data_For_Filter.png)
+
+_This is the result of executing the command `filter ft/2022-03`:_
 
 ![Filter](images/FilterCommandExample.png)
 
+**Disclaimer** : result may vary depending upon the user input.
```

Comment:
I think "results may vary depending on the current data in ExpenseExpert." sounds better, user input seems more like the current command.

Reply:
done